src/components/ChatApp/ChatApp.css

```diff
@@ -21,7 +21,7 @@ filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#183850', end
   overflow: hidden;
 }
 .app-bar{
-  background-image: url('../../images/susi-white.svg');
+  background-image: url('../../images/susi-logo-white.png');
```

Comment:
The text vs height ratio seems good to me. on increasing height the size of whole app bar will change. @rishiraj824 @isuruAb Please provide your inputs here.
Thanks